platforms/minigame/platforms/taobao-mini-game/wrapper/engine/AssetManager.js

```diff
@@ -30,7 +30,7 @@ downloader.register({
 });
 
 function loadImage (url, options, onComplete) {
-    const img = window.screencanvas.createImage();
+    const img = $global.screencanvas.createImage();
```

Comment:
The window.screencanvas property is undefined in isolated engine environments.

Reply:
What does `isolated engine environments` mean?

Reply:
The window.screencanvas property is undefined in separate engine plugin mode.